web/lib/service/analytics.ts

```diff
@@ -12,6 +12,37 @@ init(ENV_CONFIG.amplitudeApiKey ?? '', undefined, { includeReferrer: true })
 
 export { track }
 
+// Integrate Sprig
```

Comment:
I'd put this in a new file under `web/lib/service` called `sprig.ts`. Then, import it here for effects, e.g. `import './sprig'`

Reply:
You can disable typescript for the whole file with `// @ts-nocheck` at the top instead of doing it per line.